powerauth-java-server/pom.xml

```diff
@@ -259,6 +251,27 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
+            <id>test-repository</id>
+            <activation>
+                <property>
+                    <name>!useInternalRepo</name>
+                </property>
+            </activation>
+            <properties>
```

Comment:
Empty properties are not needed. It is possible to configure `maven.deploy.skip` instead of `maven-deploy-plugin`, but @romanstrobl said that it is not convention at Wultra yet.